de.lucaswerkmeister.corebird.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "de.lucaswerkmeister.corebird",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "corebird",
```

Comment:
As you are the upstream, it would be better to handle these renames in application repo than here.

Reply:
Sorry, I don’t understand – where’s the rename in this line? The [Flatpak documentation](http://docs.flatpak.org/en/latest/manifests.html) names `command` as one of the “basic properties” of a manifest. (I do agree with the `rename-icon` comment below, though.)